### PR TITLE
Some extra new lines escaped from our test

### PR DESCRIPTION
**Pull request by shaoshing**
These empty lines contain only spaces and tabs were not considered as extra lines in our test. Let's destroy them and make sure our test can capture them in the future.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "grunt-jsonlint": "^1.0.4",
     "grunt-jsxhint": "^0.6.0",
     "grunt-lintspaces": "^0.7.0",
+    "node-lintspaces": "shaoshing/node-lintspaces",
```
Comment (iwehrman) [inline package.json]: Is this extra package actually being used? Does `grunt-lintspaces`somehow pick it up?

Comment (shaoshing) [inline package.json]: yes the new package is in use. `grunt-lintspaces` relies on `node-lintspaces` and the later one does not check lines with spaces or tabs. I made a tiny fix to its regexp to address these lines.

**Comment by iwehrman** [comment]
So, is node-lintspaces a "peer dependency" here? I'm not sure I knew this was possible... maybe I did, but I didn't realize it was this easy?

**Comment by iwehrman** [comment]
Also, you should submit your regexp fix back to the main node-lintspaces project, too :smile:

**Comment by shaoshing** [comment]
I've never use `peerDependencies` before, but I looked it up and it is more likely to be used by a plugin to resolve possible dependencies conflicts with other plugins or the host. DS here is more like the host and may not getting any benefit from it. Please let me know if I am wrong :D  BTW, submiting a PR to the project sounds great!

File: package.json (hunk 1 above)
Comment (iwehrman) [inline package.json]: Well, I have no idea why this works (@mcilroy suggests `grunt` magic?) but it does seem to work. Note that I had to `rm -rf node_modules` and then `npm install` again for the change to take effect. But, hey, better linting!

**Comment by iwehrman** [comment]
@shaoshing: let us know if/when `grunt-lintspaces` accepts your PR so we can remove the extra dependency.

**Pull request merged**